Re-punctuated doc comments, especially to display double dash better;…

dpyworks opened the pull request:
… also punctuated some non-doc comments.

dpyworks committed:
Re-punctuated doc comments, especially to display double dash better; also punctuated some non-doc comments.

dpyworks commented:
Same suggested changes mentioned a couple days ago.

berryware commented on src/cl_def.rs:
```diff
@@ -45,14 +45,14 @@ impl CommandLineDef {
-    /// * Panics if the alias starts with '--' and the length is less than 4
-    /// * Panics if the alias starts with '-' and the length is not equal to 2
+    /// * Panics if the alias does not start with `-` or `--`.
+    /// * Panics if the alias starts with `-- and the length is less than 4.
```
Are you missing a tick mark on this line? `--` perhaps?

berryware commented on src/cl_def.rs:
```diff
@@ -509,19 +509,19 @@ impl OptionDef {
     ///   then this option will be considered required and will panic if a value is not specified on the
-    ///   commandline. if `value_name` is None, `default_value` is ignored.
-    /// * `description` - The description of this option. e.g. The file to be read.
+    ///   commandline. If `value_name` == `None``, `default_value` is ignored.
```
I think `None`` should be `None`

berryware reviewed (changes requested):
Thanks for the PR David. Found 2 small issues.

dpyworks committed:
docs: missing/extra backquotes; alias vec![]; no -f for value_name

dpyworks committed:
docs: file_path instead of pathname (which had been used in previous commit)

dpyworks commented:
Hi David,
I corrected the two mistakes you mentioned.
Also, my original pull request did not handle the "aliases" example correctly:
I had changed `"-n", "--negative"` to `-n`, `negative`.
In fact, I think I should have just added vec![], that is, `vec!["-n", "--negative"]`.  What do you think?
I also found "-f --filename" in some examples and changed it to use vec! as above.
Finally, the "value_name" example had "-f path".  Since that's supposed to appear within angle brackets in a help screen, I think it should be a simple name, not a callout.  So I suggested using "filepath" instead of "-f path".
